skalenetwork/skaled
# SKALED-1431 debug_getFutureTransactions call

Rebased from v3.18.0 to v3.17.2

Test sequence:
1. Add balance for your account to skaled's config.
2. Run skaled with --enable-debug-behavior-apis
3. Send transaction with too big nonce.
4. Send curl -X POST --data '{"id":4,"jsonrpc":"2.0","method":"debug_getFutureTransactions","params":[]}' <URL>
5. See your transaction.
6. Repeat the same without --enable-debug-behavior-apis, METHOD_NOT_FOUND error should be returned.
